src/Makefile.am

```diff
@@ -69,6 +69,14 @@ libephymain_la_SOURCES = \
 	ephy-session.h				\
 	ephy-shell.c				\
 	ephy-shell.h				\
+	ephy-sync-crypto.c			\
```

Comment:
Don't worry about it, I'll handle it later. Automake is not simple to learn and you have better things to worry about!